spec/unit/inspector_spec.rb

```diff
@@ -106,4 +106,23 @@ def self.name; "BarracudaInspector"; end
       expect(BarBazInspector.new.scope).to eql("bar_baz")
     end
   end
+
+  describe "#run_inspect_script" do
+    it "calls the callback with the output" do
+      system = double
+      expect(system).to receive(:run_script) do |script, options|
```

Comment:
Unused block argument - `script`. If it's necessary, use `_` or `_script` as an argument name to indicate that it won't be used.